test_scripts/API/ServiceStatusUpdateToHMI/001_1_Video_service_protected.lua

```diff
@@ -41,6 +41,9 @@ function common.onServiceUpdateFunc(pServiceTypeValue)
     { serviceEvent = "REQUEST_RECEIVED", serviceType = pServiceTypeValue, appID = common.getHMIAppId() },
     { serviceEvent = "REQUEST_ACCEPTED", serviceType = pServiceTypeValue, appID = common.getHMIAppId() })
   :Times(2)
+
```

Comment:
@GetmanetsIrina Please update description with appropriate expectation on this notification

Reply:
@dboltovskyi, please find the update in 962e6c2